libmobilecoin/src/transaction.rs

```diff
@@ -342,10 +342,19 @@ pub extern "C" fn mc_transaction_builder_create(
                     FogResolver::new(fog_resolver.0.clone(), &fog_resolver.1)
                         .expect("FogResolver could not be constructed from the provided materials")
                 });
-        // TODO: After servers are deployed that are supporting the memos,
-        // Enable recoverable transaction history by configuring an RTHMemoBuilder
-        let memo_builder = NoMemoBuilder::default();
-        let mut transaction_builder = TransactionBuilder::new(fog_resolver, memo_builder);
+        // FIXME: block version should be a parameter, it should be the latest
+        // version that fog ledger told us about, or that we got from ledger-db
+        let block_version = BlockVersion::ONE;
```

Comment:
This seems necessary to implement for building transactions with block version > 1

Reply:
yeah, but i can't change that without changing the `libmobilecoin.h` header which impacts the swift build, so usually i put fixme's without breaking the SDK teams' build